etc/profile-a-l/chafa.profile

```diff
@@ -0,0 +1,56 @@
+# Firejail profile for chafa
+# Description: chafa is an image viewer.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include chafa.local
+# Persistent global definitions
+include globals.local
+
+blacklist /usr/libexec
```

Comment:
Which distro do you use? Does it use /usr/libexec?

Just to be sure it does not break.

Reply:
ArchLinux. It did not use /usr/libexec